app/import_functions.go

```diff
@@ -1080,7 +1080,7 @@ func (a *App) importAttachment(data *AttachmentImportData, post *model.Post, tea
 	}
 	fileInfo, appErr := a.DoUploadFile(timestamp, teamId, post.ChannelId, post.UserId, file.Name(), buf.Bytes())
 	if appErr != nil {
-		mlog.Error("Failed to upload file:", mlog.Err(err))
+		mlog.Error("Failed to upload file:", mlog.Err(appErr))
```

Comment:
:facepalm:

Reply:
This is a very common mistake I am seeing in a lot of places. Wondering if a vet check can help here.